res2df/satfunc.py

```diff
@@ -252,11 +254,11 @@ def df2ecl(
             to file.
 
     Returns:
-        Generated Eclipse include string
+        Generated resdata include string
```

Comment:
Lets use `include file format` instead of `resdata include string`. Ie. `data from the dataframes in include file format`.

Reply:
Not sure about calling the generated string contents of a file of type X (this is my understanding of it now at least, may not be correct) X-format. To be honest I'm a bit fuzzy on what the original "include string" means, @berland please clarify

Reply:
It is a string that you have to dump to a file in order to INCLUDE it into your DATA-file (you can of course bypass the file if you are developing yourself). The string will include a simulator keyword and the associated records of numbers, picked from the dataframe.

What about "include file content" ?